gatsby/src/components/reviewDate/style.css

```diff
@@ -0,0 +1,4 @@
+.review-date {
```

Comment:
@carolyn-shannon Styling for the ReviewDate component is here.

Reply:
updated this to make more consistent - thx